dotcom-rendering/src/web/components/Callout/Form.tsx

```diff
@@ -0,0 +1,250 @@
+		});
+		setValidationErrors(errors);
+		return Object.keys(errors).length === 0;
+	};
+
+	const submitForm = async (form: FormDataType) => {
+		setNetworkError('');
+
+		// need to add prefix `field_` to all keys in form, required by formstack
+		const formDataWithFieldPrefix = Object.keys(formData).reduce(
+			(acc, cur) => ({
+				...acc,
+				[`field_${cur}`]: form[cur],
+			}),
+			{},
+		);
+		return fetch(submissionURL, {
+			method: 'POST',
+			body: JSON.stringify({
+				formId: formID,
+				...formDataWithFieldPrefix,
+			}),
+			headers: {
+				'Content-Type': 'application/x-www-form-urlencoded',
+			},
+		})
+			.then((resp) => {
+				if (resp.status === 201) {
+					setSubmissionSuccess(true);
+				} else {
+					setNetworkError(
+						'Sorry, there was a problem submitting your form. Please try again later.',
+					);
+				}
+			})
+			.catch((respError) => {
+				window.guardian.modules.sentry.reportError(
+					respError,
+					'callout-embed-submission',
+				);
+
+				setNetworkError(
+					'Sorry, there was a problem submitting your form. Please try again later.',
+				);
+			});
+	};
+
+	if (submissionSuccess) {
+		return (
+			<InlineSuccess>
+				Thank you, your story has been submitted successfully. One of
+				our journalists will be in touch if we wish to take your
+				submission further.
+			</InlineSuccess>
+		);
+	}
+
+	return (
+		<form
+			action="/formstack-campaign/submit"
+			method="post"
+			css={formStyles}
+			noValidate={true}
+			onSubmit={(e) => {
+				e.preventDefault();
+				const isValid = validateForm();
+				if (!isValid) {
+					const firstInvalidFormElement: HTMLInputElement =
+						document.querySelectorAll(
+							':invalid',
+						)[1] as HTMLInputElement;
+					firstInvalidFormElement.focus();
+					return;
+				}
+				void submitForm(formData);
+			}}
+		>
+			{Object.values(validationErrors).filter((err) => err !== '')
+				.length > 0 && (
+				<ErrorSummary
+					message="Some information is missing."
+					context="Please complete all required fields."
+					cssOverrides={css`
+						width: fit-content;
+					`}
+				/>
+			)}
+			{formFields.map((formField) => (
+				<div css={formFieldWrapperStyles}>
+					<FormField
+						key={formField.id}
+						format={format}
+						formField={formField}
+						formData={formData}
+						setFieldInFormData={setFieldInFormData}
+						validationErrors={validationErrors}
+					/>
+				</div>
+			))}
+			<div css={textStyles}>
+				One of our journalists will be in contact before we publish your
+				information, so please do leave contact details.
+			</div>
+			{!!networkError && <div css={errorTextStyles}>{networkError}</div>}
+			<div css={footerPaddingStyles}>
+				<Button
+					priority="primary"
+					type="submit"
+					cssOverrides={submitButtonStyles(format)}
+				>
+					Submit
+				</Button>
+				<div
+					css={css`
+						a,
+						a:hover {
+							border: 0;
+						}
+						text-align: right;
+					`}
+				></div>
```

Comment:
I might be wrong but I don't think we need this div?

Reply:
Thank you for spotting that - will remove.